scss/mixins/_pagination.scss

```diff
@@ -11,17 +11,66 @@
 
 // scss-docs-start pagination-max-items-mixin
 // Boosted mod
-// Ensure we only display `n` items:
-// - the first `n/2` items
-// - the last `n/2` items
-// - including the .active one (so maybe not the `n/2`th last item)
+// The pagination list ((m) items) should display exactly (n) items and is built in 3 parts:
+// - the begining of the list is made of the first (n/2) items
+// - the end of the list is made of the last (n/2) items (or (n/2 - 1) depending on the active element)
+// - the last part of the list is made of the active element
 @mixin pagination-max-items($pagination-max-items) {
-  display: none;
+  @if $enable-responsive-pagination {
+    $half-items: $pagination-max-items * .5;
+    display: none;
 
-  &.active,
-  &:nth-child(-n+#{$pagination-max-items * .5}),
-  &:nth-last-child(-n+#{$pagination-max-items * .5}) {
-    display: list-item;
+    // rule 0:
+    //  - adding the (n/2) first items
+    //  - adding the (n/2 - 1) last items
+    //  - adding the (n/2) last item if the list contains exactly (n) items
+    &:nth-child(-n+#{$half-items}),
+    &:nth-last-child(-n+#{$half-items - 1}),
+    &:nth-child(#{$half-items + 1}):nth-last-child(#{$half-items}) {
+      display: flex;
+    }
+
+    // Pagination with m > n:
+    //  - rule 1: adding 3 dots after the active element if situated between the first (n/2) items and the last (n/2 - 1) items
```

Comment:
I am starting to wonder if it's the right behaviour.
Imagine that I am looking for something somewhere around page 10. It may be on page 10. Or 9. Or 11.
If I have only something '... 10 ...' it means that my ony options will be to use right and left arrows, right?
Is it ok from an accessibility point of view?

Reply:
IMO, it's still accessible so it might be fine as-is. Any thoughts @Aniort ?

Reply:
Seen with @Aniort, It's fine as-is since people still have an option to go to the next/previous page. The `...` are fine too since they aren't focusable.